resources/src/topicHeader.cpp

```diff
@@ -49,6 +49,7 @@ const char* CONTROL_LOADPARAM="/kraken/control/loadparam_srv";
 
 //sensor topics
 const char* SENSOR_IMU="/kraken/sensor/imu";
+const char* SENSOR_IMU_NEW="/kraken/sensor/imu_new";
```

Comment:
@icyflame : Ahh, just realised that I had done a `git add .`, which did't take `../../resources` and stuff. :sob: 

Added them now.